src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,68 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+interface Props {
+  todo: Todo,
+  onCloseInfo: (todo: null) => void,
+}
+
+export const TodoModal: React.FC<Props> = ({
+  todo,
+  onCloseInfo,
+}) => {
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(todo.userId).then(setUser);
+  });
+
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+      {!user
```

Comment:
What will be if there is no such user on the server? Looks like the infinite loader🥲

Reply:
I fixed it by strange way but I tried to fix it with try-finally and it didn`t work, so my decision was the only idea on how to stop the loader)